shared/structures/src/Group.ts

```diff
@@ -316,6 +316,23 @@ export class Group extends AutoEncoder {
                     },
                 });
             }
+
+            if (this.settings.requirePlatformMembershipOnRegistrationDate !== null) {
+                const requirePlatformMembershipOnRegistrationDate = new Date();
```

Comment:
Voor fitlers nooit de huidige datum gebruiken, maar `{ $: '$now' }`. Als de filter dan bv. de volgende dag opgeslagen is en hergebruikt wordt, blijft de betekenis intact

Reply:
(ook voor unwrappen van filters > UI filter builders, is dit nodig)